javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,25 @@
+const days = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+function getEventWeekday(daysToEvent) {
```

Comment:
Everything is fine clean and clear code, I would only recommend to use `const` instead of `let`.